fgpyo/sam/__init__.py

```diff
@@ -239,6 +239,11 @@ def __init__(self, mode: str, ext: str) -> None:
     BAM = ("b", ".bam")
     CRAM = ("c", ".cram")
 
+    @property
+    def indexable(self) -> bool:
+        """True if the file type can be indexed, false otherwise."""
+        return self == SamFileType.BAM or self == SamFileType.CRAM
```

Comment:
Normally best practice to use identity comparison for enums:


```suggestion
        return self is SamFileType.BAM or self is SamFileType.CRAM
```